gui/src/widgets/popupwidget.cpp

```diff
@@ -0,0 +1,95 @@
+#include "widgets/popupwidget.h"
+#include "stylehelper.h"
+#include "ui_popupwidget.h"
+
+
+using namespace scopy::gui;
+
+PopupWidget::PopupWidget(QWidget* parent) :
+	ui(new Ui::PopupWidget),
+	QWidget(parent)
+{
+	ui->setupUi(this);
+
+	StyleHelper::initColorMap();
```

Comment:
The stylehelper initColorMap should not be called for every popup widget that is constructed.